drivers/SmartThings/matter-appliance/src/embedded-cluster-utils.lua

```diff
@@ -13,17 +15,27 @@ if version.api < 10 then
   clusters.TemperatureControl = require "TemperatureControl"
 end
 
+--these clusters are not available in v10 and v11
+clusters.MicrowaveOvenControl = require "MicrowaveOvenControl"
+clusters.MicrowaveOvenMode = require "MicrowaveOvenMode"
+clusters.OvenMode = require "OvenMode"
+
```

Comment:
@HunsupJung I believe these clusters are available in v11, right? So maybe this should be changed to 
```suggestion
if version.api < 11 then
  clusters.MicrowaveOvenControl = require "MicrowaveOvenControl" 
  clusters.MicrowaveOvenMode = require "MicrowaveOvenMode"
  clusters.OvenMode = require "OvenMode"
end
```

Reply:
I see now that OvenMode is not available in the lua libs version 11. The others are though, correct?
Perhaps it would make sense then to do hat I suggested above, but to leave the OvenMode cluster out with a comment specific to it in particular, since it's not all of them.